generate/model/templates/base.js.template

```diff
@@ -1,49 +1,89 @@
 import DataLoader from 'dataloader';
-import findByIds from 'mongo-find-by-ids';
+import { findByIds, queryForRoles, getLogFilename, logger, authlog, checkAuthDoc, protectFields } from 'create-graphql-server-authorization';
+const log = logger(getLogFilename());
 
 export default class TypeName {
   constructor(context) {
     this.context = context;
     this.collection = context.db.collection('typeName');
-    this.pubsub = context.pubsub;
-    this.loader = new DataLoader(ids => findByIds(this.collection, ids));
+    this.pubsub = context.pubsub;generateAuthRoleDefinition
+    const that = this;
+    try {
+      generateAuthCodeModeReadOne
+    } catch (err) { log.error(err.message); }
```

Comment:
I thought we were going to get rid of these try/catch blocks?

Reply:
removed them